environments/development/property-cde/archive/workflow.yml

```
@@ -0,0 +1,18 @@
+dag:
+  repository: moj-analytical-services/airflow-dmet-property
+  tag: v0.4.4
+  compute_profile: "general-on-demand-1vcpu-4gb"
+
+maintainers:
+  - jnayak-moj
+
+secrets:
+  - api-key
+
+iam:
+  s3_read_write:
+    - s3://mojap-land-dev/property/cde/*
```

```suggestion
    - mojap-land-dev/property/cde/*
```
Don't think this will work as supplied currently